dao/src/main/java/com/epam/esm/dao/specification/gift/GetAllGiftCertificatesSpecification.java

```diff
@@ -0,0 +1,30 @@
+package com.epam.esm.dao.specification.gift;
+
+import com.epam.esm.dao.specification.Specification;
+
+public class GetAllGiftCertificatesSpecification implements Specification {
```

Comment:
@TimurRadko The immutable class must be declared as final (So that child classes can’t be created).

Reply:
Thank for a good idea. These classes should use in a particular case, and they must be declared final. (fixed)